app/src/main/java/zapsolutions/zap/fragments/WalletFragment.java

```diff
@@ -466,6 +460,12 @@ public void onResume() {
             mExchangeRateListenerRegistered = true;
         }
 
+        if (PrefsUtil.isWalletSetup()){
+            mWalletSpinner.init();
```

Comment:
I would rather use `updateList` to expose here and refresh the list.
And keep visibility/init in oncreate 🤔

Reply:
Yes, definetly better. I changed it to use updateList() now.
What is the problem in having the visibility in onResume instead of onCreate?